backend/src/main/java/ru/hh/career/solution/professionalenvironmentandspecialization/dto/ProfessionalEnvironmentDeserializer.java

```diff
@@ -15,18 +15,19 @@ public ProfessionalEnvironmentDeserializer() {
         super(ProfessionalEnvironmentDto.class);
     }
 
+    static final ObjectMapper mapper = new ObjectMapper();
+
     @Override
     public ProfessionalEnvironmentDto deserialize(JsonParser jsonParser, DeserializationContext deserializationContext) throws IOException {
-        ProfessionalEnvironmentDto profEnv = new ProfessionalEnvironmentDto();
+        ProfessionalEnvironmentDto professionalEnvironmentDto = new ProfessionalEnvironmentDto();
         JsonNode treeNode = jsonParser.readValueAsTree();
         if (treeNode == null) {
             return null;
         }
-        profEnv.setId(treeNode.get("id").asInt());
-        profEnv.setName(treeNode.get("name").asText());
-        ObjectMapper objectMapper = new ObjectMapper();
-        SpecializationsDto[] specDTOS = objectMapper.readValue(treeNode.get("specializations").toString(), SpecializationsDto[].class);
-        profEnv.setSpecialization(Arrays.asList(specDTOS));
-        return profEnv;
+        professionalEnvironmentDto.setId(treeNode.get("id").asInt());
+        professionalEnvironmentDto.setName(treeNode.get("name").asText());
+        SpecializationsDto[] specDTOS = mapper.readValue(treeNode.get("specializations").toString(), SpecializationsDto[].class);
```

Comment:
Поменяй, пожалуйста, на камелкейс поменять
```suggestion
        SpecializationsDto[] specDtos = mapper.readValue(treeNode.get("specializations").toString(), SpecializationsDto[].class);
```
Ну и снова сокращения